modules/lib/src/main/resources/assets/js/app/view/context/ContextView.ts

```diff
@@ -105,8 +106,8 @@ export class ContextView
         this.onRemoved(() => ApplicationEvent.un(handleApplicationEvents));
 
         VersionContext.onActiveVersionChanged((contentId: string, version: string) => {
-            if (this.item.getId() === contentId && this.isVisible() && this.activeWidget === this.versionsWidgetView) {
-                this.updateActiveWidget();
+            if (this.item?.getId() === contentId && this.isVisible() && this.activeWidget === this.versionsWidgetView) {
+                this.updateActiveWidget().catch(DefaultErrorHandler.handle);
```

Comment:
Why not catch once inside `updateActiveWidget`'s return instead of repeating catch for every single call of `updateActiveWidget`?

Reply:
Right, fixed